picoquic/frames.c

```diff
@@ -2248,8 +2282,8 @@ int picoquic_skip_frame(uint8_t* bytes, size_t bytes_max, size_t* consumed,
             break;
         case picoquic_frame_type_new_connection_id:
             /* Skip the sequence */
-            byte_index += picoquic_varint_skip(&bytes[byte_index]);
```

Comment:
Couldn't somehow `consumed == 0`? And why at all `-1`?

Reply:
Consumed always includes the type byte, already included in byte index, hence -1. Rewriting for clarity, and adding comment.
~~~
ret = picoquic_skip_connection_id_frame(&bytes[byte_index-1], bytes_max, consumed);
~~~
Consumed is set to bytes_max in case of error, thus not zero.